### ISSUE TITLE
Scene switch to Previous?

### ISSUE THREAD

**Issue by koerby** [715209/nginx-obs-automatic-low-bitrate-switching#171]
Where can i config the previous scene for switch on start? It's always the same scene like "normal"

**Comment by b3ck** [comment]
so in the config you have these options:
```json
    "switchToStartingSceneOnStreamStart": false,
    "switchFromStartingSceneToLiveScene": false
```

Let's take a look at this first one:
`"switchToStartingSceneOnStreamStart"`
This means when you start streaming in OBS via the chat command `!start` it will change to the scene you have here:
```json
    "starting": "STARTING",
```

And the second one:
`"switchFromStartingSceneToLiveScene"`
This means when you start streaming from your phone to your "middleman" server (whether it be in the cloud like belabox cloud or a locally hosted one) it will change to this scene:
```json
      "normal": "LIVE",
```

That make sense?

**Comment by koerby** [comment]
Yes, this ist ok, but when my "mobile" stream get's down and reconnected, first they switch to [previous] and then to [normal], but its the same scene? where can i change the [previous] scene?

07T05:26:47.551463Z  INFO noalbs::chat::chat_handler: executed command: Start
2024-08-07T05:27:21.756229Z  INFO NOALBS:Switcher: noalbs::switcher: Switcher running
2024-08-07T05:27:22.767023Z  INFO NOALBS:Switcher: noalbs::switcher: Scene switched to [Normal] Mobile_live
2024-08-07T05:27:32.848029Z  INFO NOALBS:Switcher: noalbs::switcher: Scene switched to [Offline] Mobile_Offline
2024-08-07T05:27:38.928385Z  INFO NOALBS:Switcher: noalbs::switcher: Scene switched to **[Previous] Mobile_live**
2024-08-07T05:27:49.960887Z  INFO NOALBS:Switcher: noalbs::switcher: Scene switched to [Normal] Mobile_live

**Comment by 715209** [comment]
Hi, the previous scene is automatically set to the last used switchable scene. You can't manually set it
